packages/neo-one-smart-contract-lib/src/__tests__/ownership/TestOwnableContract.test.ts

```diff
@@ -0,0 +1,81 @@
+import { common, crypto } from '@neo-one/client-common';
+import { SmartContractAny } from '@neo-one/client-core';
+import { withContracts } from '@neo-one/smart-contract-test';
+import * as path from 'path';
+
+const RECIPIENT = {
+  PRIVATE_KEY: '7d128a6d096f0c14c3a25a2b0c41cf79661bfcb4a8cc95aaaea28bde4d732344',
+  PUBLIC_KEY: '02028a99826edc0c97d18e22b6932373d908d323aa7f92656a77ec26e8861699ef',
+};
+
+describe('Ownable', () => {
+  test('deploy + transfer', async () => {
+    await withContracts<{ testOwnable: SmartContractAny }>(
+      [
+        {
+          filePath: path.resolve(__dirname, '..', '..', '__data__', 'contracts', 'ownership', 'TestOwnable.ts'),
+          name: 'TestOwnable',
+        },
+      ],
+      async ({ client, networkName, testOwnable: smartContract, masterAccountID }) => {
+        crypto.addPublicKey(
+          common.stringToPrivateKey(RECIPIENT.PRIVATE_KEY),
+          common.stringToECPoint(RECIPIENT.PUBLIC_KEY),
+        );
+
+        const deployResult = await smartContract.deploy(masterAccountID.address, { from: masterAccountID });
+        const deployReceipt = await deployResult.confirmed({ timeoutMS: 2500 });
+        if (deployReceipt.result.state !== 'HALT') {
+          throw new Error(deployReceipt.result.message);
+        }
+
+        expect(deployReceipt.result.gasConsumed.toString()).toMatchSnapshot('deploy consumed');
+        expect(deployReceipt.result.gasCost.toString()).toMatchSnapshot('deploy cost');
+        expect(deployReceipt.result.value).toBeTruthy();
```

Comment:
from working on neotracker seems like we want to use `.toEqual(true)` over `.toBeTruthy()`. Probably since `deployReceipt.result.value` could pass if it was `1` or something. :thinking:

Reply:
@danwbyrne  This is borrowed from the tests that Alex wrote for the ICO:
https://github.com/neo-one-suite/neo-one/blob/master/packages/neo-one-smart-contract-lib/src/__tests__/TestICO.test.ts#L34

Reply:
Yes this is a new pattern we should transition to, using `toEqual(true)` over `.toBeTruthy()` for boolean checks

Reply:
+1, let's use `toEqual(true)`

Reply:
👍